script.js

```diff
@@ -0,0 +1,201 @@
+console.log('hello');
+var count= 0;
+var board= [[null, null, null],[null, null, null],[null, null, null]];
+var box = document.querySelector('#box-1');
+var box2 = document.querySelector('#box-2');
+var box3 = document.querySelector('#box-3');
+var box4 = document.querySelector('#box-4');
+var box5 = document.querySelector('#box-5');
+var box6 = document.querySelector('#box-6');
+var box7 = document.querySelector('#box-7');
+var box8 = document.querySelector('#box-8');
+var box9 = document.querySelector('#box-9');
+
+var getWinner = function() {
+  if (isXWinner()) {
+    alert ('X is the winner!');
+  }
+  else if (isOWinner()) {
+   alert ('O is the winner!');
+  }
+  else if (board[0][0] != null && board[0][1] != null && board[0][2] != null && board[1][0] != null && board[1][1] != null && board[1][2] != null && board[2][0] != null && board[2][1] != null && board[2][2] != null){
+  	alert("Tie!");
+  // return null;
+	}
+};
+
+
+function isXWinner(){
+	return winsRowX() || winsColumnX() || winsDiagonalX();
+}
+
+function isOWinner(){
+	return winsRowO() || winsColumnO() || winsDiagonalO();
+}
+
+
+var winsRowX = function(){
+	return allThreeX (board[0][0],board[0][1],board[0][2])||
+		allThreeX (board[1][0],board[1][1],board[1][2])||
+		allThreeX (board[2][0],board[2][1],board[2][2]);	
+};
+
+var winsColumnX = function(){
+	return allThreeX (board[0][0],board[1][0],board[2][0])||
+		allThreeX (board[0][1],board[1][1],board[2][1])||
+		allThreeX (board[0][2],board[1][2],board[2][2]);
+};
+
+var winsDiagonalX = function(){
+	return allThreeX(board[0][0],board[1][1], board[2][2])||
+		 allThreeX(board[2][0],board[1][1], board[0][2]);
+};
+
+var allThreeX = function(cellOne, cellTwo, cellThree) {
+
+  return (cellOne === 'x') && (cellTwo === 'x') && (cellThree === 'x');
+};
+
+var winsRowO = function(){
+	return allThreeO (board[0][0],board[0][1],board[0][2])||
+		allThreeO (board[1][0],board[1][1],board[1][2])||
+		allThreeO (board[2][0],board[2][1],board[2][2]);	
+};
+
+var winsColumnO = function(){
+	return allThreeO (board[0][0],board[1][0],board[2][0])||
+		allThreeO (board[0][1],board[1][1],board[2][1])||
+		allThreeO (board[0][2],board[1][2],board[2][2]);
+};
+
+var winsDiagonalO = function(){
+	return allThreeO (board[0][0],board[1][1], board[2][2])||
+		 allThreeO (board[2][0],board[1][1], board[0][2]);
+};
+
+var allThreeO = function(cellOne, cellTwo, cellThree) {
```

Comment:
Notice how your functions for winsRow, winsColumn, winsDiagonal, and allThree are basically the same? You could make these one function and pass O or X as an argument to check if either has won. This will make your functions more generic and your code DRYer as well.